src/fetch/trader_data.py

```diff
@@ -214,6 +235,12 @@ def fetch_trader_data(
     :param load_from: File to load from (todo - load from).
     :return: collection of `CowSwapTrader` on `network` at `block_number`
     """
+    network_file = load_from.filename(network)
+    try:
+        return CowSwapTrader.load_from(network_file)
+    except FileNotFoundError:
+        print(f"file at {network_file.name} not found. Fetching from Dune")
```

Comment:
I'm surprised we weren't already lazy loading this file (I though the large refactoring unified all fetching steps)

Reply:
I jut forgot to do it in the BIG refactor.

Reply:
Now that we have a fixed version, I suppose we could still refactor a few things as long as each build does not change the permanent files.